test/metabase/util_test.clj

```diff
@@ -132,6 +132,25 @@
           (is (= expected
                  (u/slugify s))))))))
 
+(deftest ^:parallel slugify-filename-test
+  (doseq [[group s->expected]
+          {nil
+           {"ToucanFest 2017"               "toucanfest_2017"
+            "Cam's awesome toucan emporium" "cam_s_awesome_toucan_emporium"
+            "Frequently-Used Cards"         "frequently_used_cards"}
+
+           "check that diactrics get removed"
+           {"Cam Saul's Toucannery"      "cam_saul_s_toucannery"
+            "toucans dislike piñatas :(" "toucans_dislike_pinatas___" }
+
+           "check that non-ASCII characters are preserved"
+           {"勇士" "勇士"}}]
+    (testing group
+      (doseq [[s expected] s->expected]
+        (testing (list 'u/slugify-filename s)
+          (is (= expected
+                 (u/slugify-filename s))))))))
```

Comment:
I was going to complement you on your fantastic test cases until I realized it's just a copy of the `u/slugify` test... if anything I think that's proof that this should be one function with options instead of two

Reply:
These have been merged, but I've preserved the separate test case as `slugify-unicode-test`.